opencensus/trace/exporters/stackdriver_exporter.py

```diff
@@ -162,15 +162,19 @@ def translate_to_stackdriver(self, spans):
         spans_list = []
 
         for span in spans_json:
+            # Need to convert span_id from int to hex string
```

Comment:
Can you please make this a function instead duplicating the code between span-id and parent-span-id